notebooks/official/workbench/inventory-prediction/inventory_prediction.ipynb

```diff
@@ -799,11 +752,11 @@
         "id": "0250e32de345"
       },
       "source": [
-        "Most of the fields like discount, department, distribution center-id have a decent distribution. For the field `product_category`, there are some categories that don't constitute 2% of the dataset at least. Although there are outliers in some numerical fields, they are exempted from removing as there can be products that are expensive or belonging to a particular category that doesn't often see many sales. \n",
+        "Most of the fields like discount, department, distribution center-id have a decent distribution. For the field `product_category`, there are some categories that don't constitute 2% of the dataset at least. Although there're outliers in some numerical fields, they are exempted from removing as there can be products that are expensive or belonging to a particular category that doesn't often see many sales. \n",
```

Comment:
Change back to "there are outliers" (i.e., no contraction). The "are" does have stress on it, therefore, don't contract on to "there".